pkg/datapath/loader/netlink.go

```diff
@@ -178,13 +142,22 @@ func replaceDatapath(ctx context.Context, opts replaceDatapathOptions) (_ func()
 	var ve *ebpf.VerifierError
 	if errors.As(err, &ve) {
 		if _, err := fmt.Fprintf(os.Stderr, "Verifier error: %s\nVerifier log: %+v\n", err, ve); err != nil {
-			return nil, fmt.Errorf("writing verifier log to stderr: %w", err)
+			revert()
```

Comment:
So these revert calls were missing before?

Reply:
Yep, looks like it.